subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -433,6 +443,34 @@ void ull_adv_sync_release(struct ll_adv_sync_set *sync)
 	sync_release(sync);
 }
 
+void ull_adv_sync_info_fill(struct ll_adv_sync_set *sync,
+			    struct pdu_adv_sync_info *si)
+{
+	struct lll_adv_sync *lll_sync;
+
+	/* NOTE: sync offset and offset unit filled by secondary prepare.
+	 *
+	 * If sync_info is part of ADV PDU the offs_adjust field
+	 * is always set to 0.
+	 */
+	si->offs_units = 0U;
+	si->offs_adjust = 0U;
+	si->offs = 0U;
+
+	/* Fill the interval, channel map, access address and CRC init */
+	si->interval = sys_cpu_to_le16(sync->interval);
+	lll_sync = &sync->lll;
+	memcpy(si->sca_chm, lll_sync->data_chan_map,
+	       sizeof(si->sca_chm));
+	si->sca_chm[4] &= 0x1f;
```

Comment:
I assume that `0x1F` is some kind of mask that is applied. We don't have a `#define` for that anywhere?

Reply:
SCA is encoded as 3 MS bits in the sync_info structure's channel map array. 0x1F is used as part of set operation.

Reply:
Hmm, let me see if I understand this correctly. 

The is a 5 octec (40 bit) field containing the Channel Map (ChM) and the SCA. We apply the 0x1F mask on the "highest" octet so that we set the SCA value to 0 by doing `=& 0x1F`, and then by
```
si->sca_chm[4] |= lll_clock_sca_local_get() << 5;
```
We set the 3 bit SCA value to what `lll_clock_sca_local_get()` returns. 

Is the above correct? 

Defining this mask isn't strictly necessary, but I general prefer it.